packages/engine-server/src/markdown/remark/utils.ts

```diff
@@ -955,17 +957,29 @@ export class RemarkUtils {
           root
         ) as WikiLinkNoteV4[];
         wikiLinks.forEach((linkNode) => {
-          if (linkNode.value.indexOf("/") >= 0) {
-            const newValue = _.replace(linkNode.value, /\//g, ".");
+          //lowercase the wikilink
+          let wikilinkValue = linkNode.value.toLowerCase();
```

Comment:
Can we take advantage of this util method which I think does the same thing (for cleaning up the note file name)? https://github.com/dendronhq/dendron/blob/041a7909cc17cb16f275fe6ad0e37901693e47c7/packages/common-server/src/files.ts#L38

Reply:
Let's defer this change for V2.